### PR TITLE
fix line wrapping bug #144420973

### PR DESCRIPTION
**Pull request by bobzoller**
[pivotal story](https://www.pivotaltracker.com/story/show/144420973)

### CODE REVIEW

**Comment by makebbekus** [comment]
💪 yassssss 💪

File: cmd/ranch/util/convox.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func ConvoxRunAttached(appName, process, command string, w, h int, input io.Read
 		return -1, err
 	}
 
-	return client.RunProcessAttached(appName, process, command, "", w, h, input, output)
+	return client.RunProcessAttached(appName, process, command, "", h, w, input, output)
```
Comment (serhalp) [inline cmd/ranch/util/convox.go]: lol

**Comment by serhalp** [comment]
Woooo! Thank you!

File: cmd/ranch/util/convox.go (hunk 1 above)
Comment (bobzoller) [inline cmd/ranch/util/convox.go]: 🤦‍♂️

**Comment by bobzoller** [comment]
published as `8.0.3` ... `ranch update`

File: cmd/ranch/util/convox.go (hunk 1 above)
Comment (perplexes) [inline cmd/ranch/util/convox.go]: oh ho ho ho

Comment (demands) [inline cmd/ranch/util/convox.go]: ❗️

**Pull request merged**